### DISCUSSION TITLE
Rate Limit Search Plugin?

### DISCUSSION THREAD

**Discussion by Cooldog1099** [Flexget/Flexget#3670] [Q&A] [answered]
I have been using this setup for years, but recently, torrentleech seems to have changed their rate limit speed to 1 request every 3 seconds. The log shows many instances of 429: too many requests. Is there a way to send fewer requests per second? Thanks!

**Comment by BrutuZ** [comment, marked as answer]
https://flexget.com/Plugins/domain_delay

**Reply by Cooldog1099** [reply]
Thanks so much!
